service/templates/confirm_selection.html

```diff
@@ -2,58 +2,80 @@
 
 {% block body %}
 
-{% if is_caution_title %}
-    <span class="alert-flag-hi-viz">Caution against first registration</span>
-    <br>
-  {% endif %}
-
-<h1 class="heading-xlarge collapse-bottom">Summary of title {{params.title.number}}</h1>
-
-<p class="expand-bottom">This title was last changed on <b>{{params.last_changed_datestring}} at {{params.last_changed_timestring}}</p>
-
-<div class="grid-row">
-  <div class="column-two-thirds">
-    <div class="text">
-      <div itemscope itemtype="http://schema.org/PostalAddress" class="lede">
-        {% for line in params.title.address_lines %}
-          <span>{{line}}</span><br/>
-        {% endfor %}
-      </div>
-      <br/>
+  <div class="grid-row">
+      <!-- Limit width to two-thirds -->
 
-      <dl class="definition-tabular">
-        <dt>Tenure type</dt>
-        <dd><b>{{params.title.tenure}}</b></dd>
-      </dl>
-    </div>
+      <div class="column-two-thirds">
 
-    <div class="service-notice font-xsmall">
-      <p>The <b>title register</b> contains details about the property or land, including:</p>
-      <ul class="list-bullet">
-        <li>title number</li>
-        <li>who owns it</li>
-        <li>what they paid for it (properties only, if available)</li>
-        <li>any rights of way</li>
-      </ul>
-    </div>
-    <br/>
-    <dl class="definition-tabular font-xsmall">
-      <dt>Amount to pay</dt>
-      {% autoescape false %}
-      <dd><b>{{params.price}}</b></dd>
-      {% endautoescape %}
-    </dl>
+        {% if is_caution_title %}
+          <span class="alert-flag-hi-viz">Caution against first registration</span>
+          <br>
+        {% endif %}
+        <!-- 1. Start Title information (number, address and tenure information) -->
+        <h1 class="heading-large">Get a summary of the {{ params['title']['tenure'] }} title for {{ params['title']['address_lines'][0] }}</h1>
+        <p class="lede">The summary is a cut-down version of the information Land Registry holds. It costs {{ params['price'] }}.</p>
+        <br>
+          <div class="text">
+            <p class="lede collapse-top">
+            {% if params['title']['address_lines'] %}
+              {{ '<br>'.join(params['title']['address_lines'])|safe }}
+            {% else %}
+              {{ _('Not Available') }}
```

Comment:
The <br> is added to make each line of the address go onto a new line - when there is no address there is no need to do this?